misc/open_pdks/meta.yaml

```diff
@@ -0,0 +1,44 @@
+package:
+  name: open_pdks
+  version: 1.0.274
```

Comment:
Use the git repository and tooling which pulls the version number out of git. See the yosys and vpr packages for examples.

Reply:
I was trying to follow https://conda-forge.org/docs/maintainer/adding_pkgs.html#tarballs-no-repos but I agree it's better to stick to this repo convention.

Reply:
Done